OpenSim/Analyses/JointReaction.cpp

```diff
@@ -204,7 +204,7 @@ setupProperties()
     _propertySet.append(&_onBodyProp);
 
     _inFrameProp.setName("express_in_frame");
-    _inFrameProp.setComment("Choice of frame (ground, parent, or child) in which the calculated "
+    _inFrameProp.setComment("Names of frames in which the calculated "
```

Comment:
Consider updating the comments in the old setup file to reflect the new meaning.